node/src/service.rs

```diff
@@ -37,11 +37,50 @@ use substrate_prometheus_endpoint::Registry;
 use polkadot_service::CollatorPair;
 use sc_consensus::ImportQueue;
 
+use crate::rpc::{
+	create_full_amplitude, create_full_development, create_full_foucoco, create_full_pendulum,
+	FullDeps, RpcExtension,
+};
+
+pub use amplitude_runtime::RuntimeApi as AmplitudeRuntimeApi;
+pub use development_runtime::RuntimeApi as DevelopmentRuntimeApi;
+pub use foucoco_runtime::RuntimeApi as FoucocoRuntimeApi;
+pub use pendulum_runtime::RuntimeApi as PendulumRuntimeApi;
+
+pub type AmplitudeClient =
```

Comment:
It helps a lot to define these types because they are used frequently throughout the code.